functions/src/index.ts

```diff
@@ -1,5 +1,54 @@
 import * as functions from 'firebase-functions';
+import * as nodemailer from "nodemailer";
+import * as admin from "firebase-admin";
+import { MailOptions } from 'nodemailer/lib/json-transport';
+import { emailAuth } from './.secret/emailAuth';
 
-export const helloWorld = functions.https.onRequest((request, response) => {
-  response.send("Hello from Firebase!");
+admin.initializeApp();
+
+const transporter = nodemailer.createTransport({
+  service: "gmail",
+  auth: emailAuth
+});
+
+/**
+ * Generates an email for a sponsorship request.
+ * @returns {boolean} indicates success
+ */
+export const submitSponserForm = functions.https.onCall(async (data: ISponserForm, context) => {
+  let text = `I am interested in the ${data.package} package.`;
```

Comment:
Change `text` to just be a brief summary of the form data listed out followed by their message.

It will be easier to see relevant information. It is also uncertain as to what they might put in their message so the overall email might have to from lines which could be confusing.